src/main/java/com/develhope/spring/vehicles/entity/Vehicle.java

```diff
@@ -43,4 +43,8 @@ public class Vehicle {
     private boolean isNew;
     @Enumerated(EnumType.STRING)
     private VehicleState vehicleState;
+
+    public boolean isAvailable() {
```

Comment:
Nella mia pull ho inserito due boolean "rented" e "purchased" che fanno riferimento allo status del veicolo in quanto l'enum VehicleState fa riferimento al fatto che siano acquistabili o noleggiabili... aggiungi i controlli con quello